frontend/app/src/store/history/transactions.ts

```diff
@@ -108,15 +107,16 @@ export const useTransactions = defineStore('history/transactions', () => {
       }
 
       const { taskId } = await fetchEthTransactionsTask(payload);
-      const address = parameters?.address ?? '';
+      const accounts = parameters?.accounts;
       const taskMeta = {
         title: t('actions.transactions.task.title').toString(),
-        description: address
-          ? t('actions.transactions.task.description', {
-              address
-            }).toString()
-          : undefined,
-        address
+        description:
+          accounts && accounts.length > 0
+            ? t('actions.transactions.task.description', {
+                address: accounts[0].address,
+                chain: accounts[0].evmChain
+              }).toString()
+            : undefined
```

Comment:
So with this, you will get the first account if you have multiple or non if empty?

The biggest problem I can think of is when you mix different chains and addresses but for example, you could join and show multiple.

Reply:
this is for ignoreCache=True. We always call it inside the accounts loop
```
        const refreshAddressTxs = accountsList.map(account =>
          fetchTransactionsHandler(false, { accounts: [account] }).catch(
```

So it is guaranteed that the length of the `accounts` here is 1

Reply:
so it can be either 1 or none? maybe then we can re-evaluate it as some point